Methoden.java

```diff
@@ -1,20 +1,57 @@
  
 public class Methoden {
 
-    public static int round(int zahl){
+    public static int round(int x){
         // TODO: implementiere hier deine Lösung für a)
```

Comment:
TODO-Kommentarzeilen kannst du löschen, wenn du eine Lösung fertig implementiert hast. Sie sind ein Hilfsmittel, um die eigene Arbeit besser überblicken zu können (in VSC werden TODO-Kommentare z.B. blau unter "Problems" aufgelistet, damit sie schnell gefunden werden können).